src/src/modules/core/api/experimentsApi/types.ts

```diff
@@ -29,6 +29,17 @@ export interface IExperimentData {
   creation_time: number;
 }
 
+/**
+ * interface IExperimentDataShort
+ * a shortened version of the experiment data interface
+ * used for storing selected experiments
+ * jescalada: TODO: replace to use list of IDs instead
+ */
+export interface IExperimentDataShort {
+  id: string;
+  name: string;
+}
+
```

Comment:
I decided to limit the data stored in the select form because when it gets encrypted (for the URL) it ends up getting too large... Even when selecting just a handful of metrics or experiments we still occasionally get a 431 Fields Too Large error.

I added an issue for that error as it's a pain when reloading or saving the current URL.